tests/bounded.rs

```diff
@@ -130,6 +130,44 @@ fn close() {
     assert_eq!(q.pop(), Err(PopError::Closed));
 }
 
+#[test]
+fn get_head() {
```

Comment:
Do we need the unit tests below if we already have doctests which pretty much cover the same scenarios?
Only reason I can think for leaving such testing is to that the compiler won't complain about unused methods.

Reply:
Ideally, you should write tests that cover every (or almost every) case that your code handles. If you need help figuring out what test cases aren't being covered, the [`cargo-llvm-cov`](https://github.com/taiki-e/cargo-llvm-cov) package is invaluable. Install it with `cargo install cargo-llvm-cov` and run it with `cargo llvm-cov --open` to get a report.

Reply:
Yep, makes sense, did not know about `cargo-llvm-cov`, so appreciate your reply. I ran it and I've covered most of it already. Will cover the remaining code in the next commits.

As a future PR, I could write a few lines on the contribution guide, mentioning this, `cargo fmt--all` and all other basic stuff needed to open a PR.

Reply:
I'll resolve this issue as this branch now has a function coverage for the bounded queue of 100.00%